modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java

```diff
@@ -145,28 +151,26 @@ private void ioStatGlobalPageTrackTest(boolean isPersistent) throws Exception {
         long physicalReadsCnt = physicalReads(mmgr, CACHE_GROUP, DEFAULT_CACHE_NAME, null);
 
         if (isPersistent)
-            Assert.assertTrue(physicalReadsCnt > 0);
+            Assert.assertThat(physicalReadsCnt, Matchers.greaterThan(0L));
```

Comment:
If we import `assertThat` and `greaterThan` statically, then the assertion reads as a plain text (which is probably good for readability): 'assert that phisicalReadsCnt greater than 0`

Reply:
Fix it